src/main/java/net/coderbot/iris/pipeline/DeferredWorldRenderingPipeline.java

```diff
@@ -151,7 +152,7 @@ public DeferredWorldRenderingPipeline(ProgramSet programs) {
 
 		this.sunPathRotation = programs.getPackDirectives().getSunPathRotation();
 
-		PackShadowDirectives shadowDirectives = programs.getPackDirectives().getShadowDirectives();
+		this.shadowDirectives = programs.getPackDirectives().getShadowDirectives();
```

Comment:
Instead of storing this you might want to construct a ClearPass object for shadowtex0.